models/order.rb

```diff
@@ -0,0 +1,7 @@
+require "sequel"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.